nav2_waypoint_follower/src/waypoint_follower.cpp

```diff
@@ -175,18 +175,24 @@ WaypointFollower::followWaypoints()
 
   rclcpp::WallRate r(loop_rate_);
 
+  auto start_time = steady_clock_.now();
+  auto last_wp_time = start_time;
+  rclcpp::Duration elapsed_time{0, 0};
```

Comment:
This isn't used, you reinstantiate a `auto elapsed_time = now_time - start_time;` in L188.

Reply:
Done in 41d480d4a85169076efe88cd9ff85b67cd5493e3